tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -0,0 +1,442 @@
+/* test_broadcast_reception.c - unit test for broadcast reception */
+
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/fff.h>
+
+#include "bluetooth.h"
+#include "cap_commander.h"
+#include "conn.h"
+#include "expects_util.h"
+#include "cap_mocks.h"
+#include "test_common.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(bt_broadcast_reception_test, CONFIG_BT_CAP_COMMANDER_LOG_LEVEL);
+
+#define FFF_GLOBALS
+
+#define ADDRESS 0x123456
+#define SID          0x0E
+#define ADV_INTERVAL 10
+#define BROADCAST_ID 0x55AA55
```

Comment:
Do we need defines for these? Aren't/can't they just be hardcoded values?

`ADDRESS` isn't even used :)

Reply:
Removed address; otherwise I am not a big fan of hardcoded values

Reply:
These are still hardcoded values :P It doesn't really make much difference if you define them here or just write them once. 

```c
#define A_VAL 10
int a = A_VAL;
```
and
```c
int a = 10;
```
Is the same IMO. If you used the the defines multiple places or in function class where it's less obvious, then I would agree with you. 

But it is OK to keep, but does not add much value IMO